modules/serverless-logic-install-kn-workflow-binary-file-linux.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/serverless-logic-install-kn-workflow-plugin-cli.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-kn-workflow-binary-file-linux_{context}"]
+= Installing the OpenShift Serverless Logic Knative Workflow plugin for Linux
+
+If you are using a Linux distribution that does not have RPM or another package manager installed, you can install the Knative Workflow plugin.
+
+.Prerequisites
+
+* You have installed the Knative (`kn`) command-line interface (CLI).
+
+* If you are not using {op-system-base} or Fedora, ensure that the *libc* library is installed in a directory on your path.
++
+[IMPORTANT]
+====
+If the *libc* library is not available, you might see the following error when you run commands:
+
+[source,terminal]
+----
+$ kn: No such file or directory
+----
+====
+
+.Procedure
+
+. Download the latest `tar` archive suitable for your environment, from the link:https://mirror.openshift.com/pub/cgw/serverless-logic/latest/[Serverless Logic download mirror] page.
+
+. Extract the Knative Workflow plugin binary file by running the following command:
++
+[source,terminal]
+----
+$ tar xvzf <tar_archive>
+----
+
+. Rename the extracted Knative Workflow plugin binary file to `kn-workflow` by running the following command:
++
+[source,terminal]
+----
+$ mv <filename> kn-workflow
+----
+
+. Install the `kn-workflow` command as the Knative CLI plugin.
```

Comment:
IBM SG says use colons leading into substeps:
```suggestion
. Install the `kn-workflow` command as the Knative CLI plugin:
```

Reply:
Updated wherever it is needed